common/persistence/sql/sqlplugin/postgresql/db.go

```diff
@@ -119,9 +119,9 @@ func (pdb *db) DbName() string {
 func (pdb *db) ExpectedVersion() string {
 	switch pdb.dbKind {
 	case sqlplugin.DbKindMain:
-		return postgresqlschema.Version
+		return postgresqlschemaV96.Version
```

Comment:
Will this cause issues for users who upgrade?

Reply:
I don't think so. When they upgrade, they'll need to update the plugin name to "postgres12", which will make use the `db_v12.go`.